stubs/redis/__init__.pyi

```diff
@@ -0,0 +1,6 @@
+# StrictRedis is defined as attribute in redis and mypy does not allow
+# imports of attributes. But to define stubs of redis we have to import
+# it. Thus to avoid mypy error, ignore is added here. 
+from redis import StrictRedis as RedisStrict  # type: ignore[attr-defined]
+
+StrictRedis = RedisStrict
```

Comment:
The stubs of redis library in typshed are a little ambiguous because:
* it does not define annotations for some methods.
* For some methods it does not provide strict typing ( just uses Any type ).

To overcome this issue, we can use redis library's code which is now mostly typed for example:
* method like `mget` is not type annotated in stubs of typshed, but this method is fully annotated in library's code ( https://github.com/redis/redis-py/blob/e5e265de87dfe7ef8cc5cad8c247d067c74fa44d/redis/commands/core.py#L1780 ). Similar goes for   `mset` and `hset` methods.

Reply:
Makes sense.